pages/apim/3.x/event-native/apim-event-native-api-management-activate-mode.adoc

```diff
@@ -0,0 +1,13 @@
+[[apim-event-native-api-management-activate-mode]]
+= Activating/deactivating the new execution engine mode
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_event_native_api_management_activate_mode.html
+:page-folder: apim/event-native
+:page-layout: apim3x
+
+[label label-version]#New in version 3.19.0#
+
+To activate the new policy execution engine mode, you must enable it in the `gravitee.yml` file using the following new keys:
```

Comment:
Previously, it is explain that we allow moving to the new execution engine in a per api basis but I don't see this part explain here. We should probably add some screenshots do guide the user on how he can enable or disable the new engine on its apis. Also I would expect some explanation on how it is easy to achieve so I would be reassured. Another thing that could be added here is a complete guide on how to test the new engine with the debug mode and we could show that the user is completely free to test it and check that everything look good prior to really enable the new mode and deploy its api.